src/main.rs

```diff
@@ -0,0 +1,58 @@
+use clap::{arg, value_parser, Command};
+
+fn cli() -> Command {
+    Command::new("state-reconstruct")
+        .about("zkSync state reconstruction tool")
+        .subcommand_required(true)
+        .arg_required_else_help(false)
+        .subcommand(
+            Command::new("import")
+                .about("Import state")
+                .subcommand_required(true)
+                .subcommand(
+                    Command::new("l1")
+                        .about("Import state from Ethereum L1")
+                        .arg(
+                            arg!(--"start-block" <START_BLOCK>)
+                                .help("Ethereum block number to start state import from")
+                                .default_value("16627460")
```

Comment:
```suggestion
                                .default_value(ZKSYNC_GENESIS_BLOCK)
```
The default value should be a constant.